pull_request_template.md

```diff
@@ -1,7 +1,11 @@
+## Reminders
+- [ ] Did I review my own code.
+- [ ] Is there only 1 commit.
+
 ## Checklist before requesting a review
 - [ ] I have performed a self-review of my code
 - [ ] I have added thorough tests. (If tests are not applicable, explain why in the next section.)
 
```

Comment:
Do you see duplicates here? Maybe we can combine this. Also, did you use do these checks before asking for a review?

Reply:
ooh I can add it into the 'checklist before requesting a review' then remove 'Did I review my own code', since 'I have performed a self-review of my code'?